src/ontology/cl-edit.owl

```diff
@@ -21969,16 +21981,6 @@ AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009019 "cortical nephrogeni
 AnnotationAssertion(rdfs:label obo:CL_0009019 "nephrogenic zone cell")
 EquivalentClasses(obo:CL_0009019 ObjectIntersectionOf(obo:CL_0002681 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0009871)))
 
-# Class: obo:CL_0009018 (lymphocyte of large intestine lamina propria)
-
-AnnotationAssertion(obo:IAO_0000115 obo:CL_0009018 "A lymphocyte that resides in the lamina propria of the large intestine.")
-AnnotationAssertion(<http://purl.obolibrary.org/obo/pato#location_grouping> obo:CL_0009018 "")
-AnnotationAssertion(oboInOwl:created_by obo:CL_0009018 <http://orcid.org/0000-0002-2825-0621>)
-AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0009018 "https://orcid.org/0000-0003-4183-8865")
-AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0009018 "lamina propria lymphocyte of large intestine")
-AnnotationAssertion(rdfs:label obo:CL_0009018 "lymphocyte of large intestine lamina propria")
-EquivalentClasses(obo:CL_0009018 ObjectIntersectionOf(obo:CL_0000542 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0011189)))
-
```

Comment:
Something odd going on here with diffs.  Ignore for now as may disappear.

Reply:
Yeah, I noticed it too, but it seems to add it back too? So I guess it's fine?